modules/adagioAnalyticsAdapter.js

```diff
@@ -257,11 +278,33 @@ function handlerBidResponse(event) {
     return;
   }
 
+  if (!event.pba) {
+    return;
+  }
+
   cache.updateAuction(event.auctionId, event.adUnitCode, {
-    adg_sid: event.seatId || null
+    ...addKeyPrefix(event.pba, 'e_')
   });
 };
 
+function handlerAuctionEnd(event) {
+  const { auctionId } = event;
+
+  if (!guard.auctionTracked(auctionId)) {
+    return;
+  }
+
+  const adUnitCodes = cache.getAllAdUnitCodes(auctionId);
+  adUnitCodes.forEach(adUnitCode => {
+    const mapper = (bidder) => event.bidsReceived.find(bid => bid.adUnitCode === adUnitCode && bid.bidder === bidder) ? '1' : '0';
```

Comment:
When we have duplicated adUnit codes - do you agree if one bidder reply a bid for one of them but not for both - we'll still track a response from the bidder for both?